compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -626,24 +627,39 @@ areNotFullTiles(ArrayRef<int64_t> inputShape,
   return false;
 }
 
+static SmallVector<OpFoldResult> getMixedValues(MLIRContext *context,
```

Comment:
Are we able to reuse the upstream method?

https://github.com/llvm/llvm-project/blob/3dd73dc1996940645620fd191110b57c49183531/mlir/include/mlir/Dialect/Utils/StaticValueUtils.h#L122